tests/suites/static_analysis/textprimarykey.sh

```diff
@@ -0,0 +1,29 @@
+run_text_primary_key() {
+	# Due to a bug in old versions of SQLite, the devs decided that non-integer
+	# primary keys can take a null value by default. We never want this. This
+	# test ensures that there are no tables in our DDL which could be affected
+	# by this.
+	# See https://www.sqlite.org/lang_createtable.html
+	res="$(grep -r 'TEXT PRIMARY KEY' ./domain/schema/)"
```

Comment:
does `TEXT PRIMARY KEY NOT NULL` would fail then?

Reply:
Resolved